modules/apps/portal-search/portal-search/src/test/java/com/liferay/portal/search/internal/buffer/IndexerRequestBufferHandlerTest.java

```diff
@@ -130,6 +137,9 @@ private void _deepReindex() throws Exception {
 			indexerRequest, _indexerRequestBuffer);
 	}
 
+	private static final MockedStatic<FrameworkUtil>
```

Comment:
I'm not sure if will need the `MockedStatic` anymore, Could you please check that?